finders/schemas/countryside-stewardship-grants.json

```diff
@@ -1,5 +1,67 @@
 {
   "slug": "countryside-stewardship-grants",
   "document_noun": "grant",
-  "facets": []
+  "facets": [
+    {
+      "key": "grant_type",
+      "name": "Grant type",
+      "type": "multi-select",
+      "preposition": "of type",
+      "allowed_values": [
+        {"label": "Option", "value": "option"},
+        {"label": "Capital", "value": "capital"}
+      ]
+    },
+    {
+      "key": "land_use",
+      "name": "Land use",
+      "type": "multi-select",
+      "preposition": "for",
+      "allowed_values": [
+        {"label": "Access", "value": "access"},
+        {"label": "Arable land", "value": "arable-land"},
+        {"label": "Boundaries", "value": "boundaries"},
+        {"label": "Coast", "value": "coast"},
+        {"label": "Farmland wildlife", "value": "farmland-wildlife"},
+        {"label": "Flood risk", "value": "flood-risk"},
+        {"label": "Grassland", "value": "grassland"},
+        {"label": "Historic environment", "value": "historic-environment"},
+        {"label": "Livestock management", "value": "livestock-management"},
+        {"label": "Organic land", "value": "organic-land"},
+        {"label": "Trees (non-woodland)", "value": "trees-non-woodland)"},
+        {"label": "Uplands", "value": "uplands"},
+        {"label": "Vegetation control", "value": "vegetation-control"},
+        {"label": "Water quality", "value": "water-quality"},
+        {"label": "Wildlife package", "value": "wildlife-package"},
+        {"label": "Woodland", "value": "woodland"}
+      ]
+    },
+    {
+      "key": "tiers_or_standalone_items",
+      "name": "Tiers or standalone items",
+      "type": "multi-select",
+      "preposition": "for",
+      "allowed_values": [
+        {"label": "Higher Tier", "value": "higher-tier"},
+        {"label": "Mid Tier", "value": "mid-tier"},
+        {"label": "Standalone capital items", "value": "standalone-capital-items"}
```

Comment:
Have we found out whether it's only possible to have certain combinations of these allowed values and the allowed values for grant type? I think that's something Holly was asking about.